src/components/_cards/BondingTableCard.tsx

```diff
@@ -37,32 +37,74 @@ const BondingTableCard: VFC<TableProps> = (props) => {
     useAaveStaker()
   const { doHandleTransaction } = useHandleTransaction()
   const { userStakes, claimAllRewards } = userStakeData
+  const [unbondLoading, setUnbondLoading] = useState<Set<number>>(
```

Comment:
Oh this is interesting! I've not come across `Set` before. I guess it's an array/map? 👀

Reply:
Set is like an array, you can store anything to it which has built in properties and methods(ex: `.add`, `.has`, `.size`, etc) that perfect for this use case. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set 
And it's more performant than array 
array vs Set benchmark : https://stackoverflow.com/a/46190569

for `map` javascript has different objects:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map